.circleci/config.yml

```diff
@@ -1,6 +1,47 @@
-version: 2
+version: 2.1
+
+orbs:
+  aws-s3: circleci/aws-s3@3.0.0
+
+commands:
+  compile-upload-tool:
+    parameters:
+      os:
+        type: string
+      tool:
+        type: string
+
+    steps:
+      - checkout
+      - setup_remote_docker:
+          version: 20.10.11
+          docker_layer_caching: true
+      - run:
+          name: Download compile_version_upload_s3.sh script
+          command: |
+            wget https://raw.githubusercontent.com/readthedocs/readthedocs.org/humitos/build-tool-no-upload-production/scripts/compile_version_upload_s3.sh
+      - run:
+          name: Download Docker image
+          command: |
+            docker pull readthedocs/build:<< parameters.os >>
+      - run:
+          name: Compile tool version
+          command: |
+            ./compile_version_updload_s3.sh << parameters.tool >>
+      - aws-s3/copy:
+          aws-region: $AWS_REGION
+          aws-access-key-id: $AWS_ACCESS_KEY_ID
+          aws-secret-access-key: $AWS_SECRET_ACCESS_KEY
```

Comment:
Just fyi, these aws-s3 parameters are the *names* of the environment variables, not the actual env var values. For example, this is the config from the docs:

```
aws-access-key-id: AWS_ACCESS_KEY_ID_BLUE
aws-region: AWS_REGION_BLUE
aws-secret-access-key: AWS_SECRET_ACCESS_KEY_BLUE
```

so we'd probably want something like:

```
      - run:
          name: Compile tool version
          command: |
            ./compile_version_upload_s3.sh << parameters.tool >>
      - aws-s3/copy:
          aws-region: ORG_AWS_REGION
          aws-access-key-id: ORG_AWS_ACCESS_KEY_ID
          aws-secret-access-key: ORG_AWS_SECRET_ACCESS_KEY
      - aws-s3/copy:
          aws-region: COM_AWS_REGION
          aws-access-key-id: COM_AWS_ACCESS_KEY_ID
          aws-secret-access-key: COM_AWS_SECRET_ACCESS_KEY
```

And then both sets of secrets are defined in the CircleCI env var UI.